addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -401,7 +401,10 @@ def _import_fattura_pa(self, tree, invoice):
             company = elements and self.env['res.company'].search([('vat', 'ilike', elements[0].text)], limit=1)
             if not company:
                 elements = tree.xpath('//CessionarioCommittente//CodiceFiscale')
-                company = elements and self.env['res.company'].search([('l10n_it_codice_fiscale', 'ilike', elements[0].text)], limit=1)
+                if not elements:
+                    _logger.warning('No Codice Fiscale found in the xml.')
+                    continue
+                company = self.env['res.company'].search([('l10n_it_codice_fiscale', 'ilike', elements[0].text)], limit=1)
                 if not company:
                     # Only invoices with a correct VAT or Codice Fiscale can be imported
                     _logger.warning('No company found with VAT or Codice Fiscale like %r.', elements[0].text)
```

Comment:
Wait, so this is just failing silently? (from the user PoV)
Doesn't `account_edi` have a warning/error management mechanism?

Reply:
The warning/error management you're refering to is only available when generating an edi attachment, not for import, if I'm not missing sth :slightly_smiling_face:. I'll open a PR and log the messages in the chatter (I discussed it with Josse).